pkgs/development/python-modules/mariadb/default.nix

```diff
@@ -26,6 +28,6 @@ buildPythonPackage rec {
     description = "MariaDB Connector/Python";
     homepage = "https://github.com/mariadb-corporation/mariadb-connector-python";
     license = licenses.lgpl21Only;
```

Comment:
The license seems to be lgpl21Plus.
https://github.com/mariadb-corporation/mariadb-connector-python/tree/v1.1.10?tab=readme-ov-file#license